controllers/object.js

```diff
@@ -4,6 +4,8 @@ const formidable = require('formidable');
 const utilities = require('../libraries/utilities');
 const {fileExists, unlinkIfExists, mkdirIfNotExists} = utilities;
 const {startSplatTask} = require('./object/SplatTask.js');
+const server = require('../server');
```

Comment:
I don't think this kind of circular import is well-liked by the server but I might be wrong here